examples/webgl_materials_envmaps_hdr.html

```diff
@@ -146,7 +144,7 @@
 				};
 
 				var hdrUrls = genCubeUrls( "../examples/textures/cube/pisaHDR/", ".hdr" );
-				new THREE.HDRCubeTextureLoader().load( hdrType, hdrUrls, function ( hdrCubeMap ) {
+				new THREE.HDRCubeTextureLoader().load( THREE.UnsignedByteType, hdrUrls, function ( hdrCubeMap ) {
```

Comment:
Ok, good to know.

Switching it to FloatType produces errors on Windows (ANGLE maybe?). FloatType works fine on OSX.

`Error: WebGL: generateMipmap: Texture at base level is not unsized internal format or is not color-renderable or texture-filterable.`

![blackuv](https://cloud.githubusercontent.com/assets/1383811/13503068/a15bc64c-e13b-11e5-80ad-1748b4ad53d9.PNG)